src/screens/NewsScreen/Search.js

```diff
@@ -82,21 +82,21 @@ const styles = StyleSheet.create({
     paddingTop: 0,
     paddingBottom: 0,
     alignSelf: 'center',
-    color: COLORS.gray1
+    color: COLORS.blackgrey
   },
   wrapperIcon: {
     marginLeft: 8,
     alignSelf: 'center',
     justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holytosca,
+    color: COLORS.anon_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.holytosca` to `COLORS.anon_primary` seems to be part of the color theme enhancement. As with the previous comment, without knowing the exact colors represented by these variables, it's hard to say whether this change is an improvement or not.
```diff
-    color: COLORS.holytosca,
+    color: COLORS.anon_primary,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->